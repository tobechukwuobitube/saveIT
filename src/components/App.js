import React from "react";
import { Route, Switch } from "react-router-dom";
import LandingPage from './LandingPage/LandingPage';


function App() {
  return (
    <div>
      <Switch>
        <Route exact path="/" component={LandingPage} />
      </Switch>
    </div>
  );
}

export default App;