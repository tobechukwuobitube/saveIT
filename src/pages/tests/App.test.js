import React from 'react';
import { Provider } from 'react-redux';
import store from '../store';
import renderer from 'react-test-renderer';
import App from '../';

describe('App', () => {
  it('should render without crashing', () => {
    const wrapper = renderer.create(
      <Provider store={store}>
        <App />
      </Provider>
    );
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
