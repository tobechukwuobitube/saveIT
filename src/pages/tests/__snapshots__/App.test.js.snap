// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render without crashing 1`] = `
<div>
  <div
    id="notification-wrapper"
  />
  <div
    id="header"
  >
    <div
      id="container"
    >
      <div
        className="navigation"
      >
        <a
          className="app-logo"
          href="/"
          onClick={[Function]}
        >
          <p
            className="logo"
          >
            Banka
          </p>
        </a>
        <nav
          id="nav"
        >
          <ul>
            <li>
              <a
                className="btn"
                href="/login"
                onClick={[Function]}
              >
                Login
              </a>
            </li>
          </ul>
        </nav>
      </div>
      <div
        className="header-caption"
      >
        <div
          className="caption-contents"
        >
          <h1>
            Banka Money Services
          </h1>
          <p>
            Manage Your Money with Ease
          </p>
          <a
            className="call-to-action"
            href="/signup"
            onClick={[Function]}
          >
            Bank with us
          </a>
        </div>
      </div>
    </div>
    <footer
      id="footer"
    >
      <p>
        Tobechukwu Obitube Â© 2019.
      </p>
    </footer>
  </div>
</div>
`;
